src/mmw/apps/user/tests.py

```diff
@@ -97,3 +99,15 @@ def test_no_token_bad_credentials_returns_400(self):
         self.assertEqual(response.status_code, 400,
                          'Incorrect server response. Expected 400 found %s'
                          % response.status_code)
+
+
+class ItsiSignupTestCase(TestCase):
+
+    def test_small_name(self):
+        username = trim_to_valid_length('shortname', '.itsi')
+        self.assertLessEqual(len(username), 30)
```

Comment:
Consider adding a second assert to ensure that `username` is "shortname.itsi". The concatenation isn't very obvious given that the function is named `trim_to_valid_length`, so it would be valuable to have a test to communicate and verify that behavior.